jupyter-book/deconvolution/deconv_covid_MuSiC_v0.ipynb

```diff
@@ -0,0 +1,2416 @@
+{
```

Comment:
<p>Can you add a sentence why this method requires a dense matrix? and that you are transforming to dense here. Also you are using now the raw counts matrices right? I assume because Music requires specific processing. Worth to mention that we overwrite X with the raw counts here again </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/cross-modal-single-cell-best-practices/pull/93/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>I'm actually not sure why we need this. I think it's a relic from a previous code.</p><p>We will check that.</p>